generator/src/routes/+page.svelte

```diff
@@ -61,33 +86,54 @@
         </div>
       </div>
       <div class="actions-wrapper">
-        <Button title="Generate IOXIO Tag" onClick={() => {}} />
+        <Button
+          disabled={status === Status.GENERATING}
+          title="Generate IOXIO Tag"
+          onClick={() => {
+            status = Status.GENERATING
+          }}
+        />
       </div>
     </form>
   </div>
   <!-- Right Panel -->
   <div class="qrcode-area-wrapper">
     <div class="qrcode-area">
       <div class="qrcode">
-        <img class="frame" src={SubtractSvg} alt="" aria-hidden="true" />
+        {#if status !== Status.GENERATING}
+          <img class="frame" src={SubtractSvg} alt="" aria-hidden="true" />
+          <img class="logo" src={LogoTagsSvg} alt="" aria-hidden="true" />
+        {:else}
```

Comment:
Why would the logo not be shown during generating? It is in the design.

Reply:
<img width="594" alt="image" src="https://github.com/ioxiocom/ioxio-tags/assets/137529618/5920c78c-8276-4283-bef0-0db085774884">